src/fallbackError.ts

```diff
@@ -75,4 +76,18 @@ export class FallbackErrorDecoder {
     };
     return result;
   }
+
+  // Construct an Error from a StatusObject.
+  // Adapted from https://github.com/grpc/grpc-node/blob/master/packages/grpc-js/src/call.ts#L79
+  callErrorFromStatus(status: DecodedRpcStatus): Error {
+    const message = `${status.code} ${Status[status.code]}: ${status.message}`;
+    delete status.message;
+    return Object.assign(new Error(message), status);
```

Comment:
This line of code is confusing to me.  It looks like you're smashing a `DecodedRpcStatus` onto an `Error` object, but not changing the type.  Should we return a type here that reflects the properties you're returning?

Reply:
Yeah, it's `ServiceError` from gRPC, I will import and use it here.

Reply:
Well no, I don't have any `Metadata` so I cannot use the type from gRPC but I'm now exporting my own (similar) type.